src/services/services.ts

```diff
@@ -944,6 +944,10 @@ namespace ts {
         }
     }
 
+    export interface HostCancellationToken {

```

Comment:
No.  It just represents the API we need from the host for the cancellation tokens it provides.